aidial_adapter_openai/app.py

```diff
@@ -172,14 +184,25 @@ async def embedding(deployment_id: str, request: Request):
     api_base, upstream_deployment = parse_upstream(
         request.headers["X-UPSTREAM-ENDPOINT"], ApiType.EMBEDDING
     )
+    api_version = get_api_version(request)
+
+    if api_version == "":
+        raise HTTPException(
+            "Api version is a required query parameter",
+            400,
+            "invalid_request_error",
+        )
+
+    if api_version in api_versions_mapping:
```

Comment:
Why do we need this?
`get_api_version` should already do the `api_versions_mapping` transformation.

Reply:
removed, thank you